ee/vellum_ee/workflows/display/workflows/vellum_workflow_display.py

```diff
@@ -293,8 +298,12 @@ def _generate_entrypoint_display(
         )
 
         entrypoint_target = get_unadorned_node(entrypoint)
-        target_node_id = node_displays[entrypoint_target].node_id
-        target_handle_id = node_displays[entrypoint_target].get_target_handle_id()
```

Comment:
note that this means even before this PR, we don't support an entrypoint feeding into a merge node. Which is dumb but still valid